types/fast-decode-uri-component/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts"
```

Comment:
pls add a test

Reply:
You still need to add test file name here . Please follow https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#create-a-new-package when adding new package